client/blocks/plugins-update-manager/schedule-form.tsx

```diff
@@ -227,7 +229,7 @@ export const ScheduleForm = ( props: Props ) => {
 								onBlur={ () => setFieldTouched( { ...fieldTouched, timestamp: true } ) }
 							></RadioControl>
 							{ frequency === 'weekly' && (
-								<Flex gap={ 6 }>
+								<Flex gap={ 6 } className={ classnames( { 'is-mobile': isMobile } ) }>
```

Comment:
Following [the Flex component documentation](https://wordpress.github.io/gutenberg/?path=/docs/components-flex--docs), we can achieve this by adding property direction property.

`<Flex gap={ 6 } direction={ [ 'column', 'row' ] }>`

Reply:
TIL! Thanks! I'll change it.